SubZero/src/model/Model.h

```diff
@@ -8,24 +8,46 @@
 #ifndef MODEL_H_
 #define MODEL_H_
 
+#include <string>
+
+using namespace std;
+
 /**
  * This is an abstract class that represents the physical attributes of the sub.
  * It accepts input from cameras and FPGA and outputs to FPGA and
  * cameras (strictly speaking). Model contains references to the tools required
  * for the assets to run. Each Model inherits from observable and has a state.
  * Each Model has a filter.
+ *
+ * @author	Zack Tzeng
+ * @version	0.0
+ * @since	2014-03-01
  */
-
 class Model {
+
+private:
+	FilterManager* filterManager;
+	FilterFactory* filterFactory;
```

Comment:
No need for a reference to FilterFactory since all of its functions will be static